ngraph/core/src/op/softplus.cpp

```diff
@@ -29,6 +29,13 @@ bool op::v4::SoftPlus::visit_attributes(AttributeVisitor& visitor)
 void op::v4::SoftPlus::validate_and_infer_types()
 {
     NGRAPH_OP_SCOPE(v4_SoftPlus_validate_and_infer_types);
+    const element::Type& input_et = get_input_element_type(0);
```

Comment:
Can you add more tests (for visit_attributes)?
https://github.com/openvinotoolkit/openvino/tree/master/ngraph/test/visitors/op

Reply:
there is no attribute for this op.  the "threshold" we discussed before is an internal defined parameter, which is not exposed to user.  is it ok not to add test for visit_attributes?

Reply:
Yes we need to add tests for such operations too.

Example you can find here: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/visitors/op/result.cpp

Reply:
thanks for reminder, I have added it, please help check it.

Reply:
@ilyachur is this PR ok to be merged into master?